### PR TITLE
CALS-5696 Removing (required) label from Employment Card

### PR DESCRIPTION
**Pull request by nasar-cambria**
#### Which User story does this relate to?
- https://osi-cwds.atlassian.net/browse/CALS-5696

#### Brief feature description
- Since the (required) label in address fields are coming from wood-duck its adding (required) label to every common address component in RFA 01A. We need to make sure this label is only added to fields which are required.

#### Does this PR block/effect other stories?
- no

#### Do any commands needs to be run in order to run the project with this PR?
- no

#### Code coverage percent diff as a result of this PR?
- NA

### CODE REVIEW

File: app/javascript/rfa_forms/rfa01a_edit_view/referencesCard.jsx [hunk 1]
```diff
@@ -53,6 +53,7 @@ export default class ReferencesCard extends React.Component {
             index={this.props.index}
             stateTypes={this.props.stateTypes}
             addressTitle='Physical Address (required)'
```
Comment (anuroop15) [inline app/javascript/rfa_forms/rfa01a_edit_view/referencesCard.jsx]: why there are two required  texts? Both addressTitle and label attributes.

File: app/javascript/rfa_forms/rfa01b_edit_view/applicantDetailsCard.js [hunk 2]
```diff
@@ -79,6 +79,7 @@ export default class ApplicantDetailsCard extends React.Component {
             stateTypes={this.props.stateTypes}
             addressTitle={'Residence Address' + RfaCommon.requiredIndicator}
             id='street_address'
+            label=' (required)'
```
Comment (anuroop15) [inline app/javascript/rfa_forms/rfa01b_edit_view/applicantDetailsCard.js]: can we use "RfaCommon.requiredIndicator" as we are already using in the file.

File: app/javascript/rfa_forms/rfa01a_edit_view/referencesCard.jsx (hunk 1 above)
Comment (nasar-cambria) [reply app/javascript/rfa_forms/rfa01a_edit_view/referencesCard.jsx]: addressTitle is for Physical address and Label is being passed as props which is coming from wood-duck for ```City``` ```State``` and ```Zip Code```

File: app/javascript/rfa_forms/rfa01b_edit_view/applicantDetailsCard.js (hunk 2 above)
Comment (nasar-cambria) [reply app/javascript/rfa_forms/rfa01b_edit_view/applicantDetailsCard.js]: As discussed in the earlier PR we would only want to use constant for lengthy labels/texts until we decide on implementing all the labels using that approach. So I would like to keep this as is for now.

Comment (anuroop15) [reply app/javascript/rfa_forms/rfa01b_edit_view/applicantDetailsCard.js]: Agreed for lengthy labels. But required case was different. We created a constant 'requiredIndicator' expecting a removal of required inputs in future.

Comment (nasar-cambria) [reply app/javascript/rfa_forms/rfa01b_edit_view/applicantDetailsCard.js]: I think creating constants leads to unnecessary render  ```https://www.kirupa.com/react/avoiding_unnecessary_renders.htm``` 

If we are thinking of an approach where we can make a change at one place and not on all the fields then having this inside of CSS would be better. Thoughts ?

Comment (ratneshraval) [reply app/javascript/rfa_forms/rfa01b_edit_view/applicantDetailsCard.js]: @nasar-cambria I don't think having a constant will cause an unnecessary render.
Although, I like your idea about keeping it in CSS so styles can be changed easily with text. But I don't think that work is needed yet, so current approach is fine with me.

**Comment by nasar-cambria** [comment]
jenkins re-test this please

**Comment by sunilb0575** [comment]
jenkins, retest this please

**Pull request merged**